src/plugins/intel_cpu/src/nodes/dft.cpp

```diff
@@ -75,6 +115,7 @@ DFT::DFT(const std::shared_ptr<ngraph::Node>& op, const dnnl::engine& eng, Weigh
     }
 
     inverse = std::dynamic_pointer_cast<ngraph::opset7::DFT>(op) == nullptr;
```

Comment:
Please replace *std::dynamic_pointer_cast<ngraph::opset7::DFT>(op) == nullptr* to *ov::is_type<op::v7::DFT>(op)*

Reply:
Done